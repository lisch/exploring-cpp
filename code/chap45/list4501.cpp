#include "list4501.cc"
