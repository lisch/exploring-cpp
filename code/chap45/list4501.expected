{ 1 2 3 3 2 1 42 }
Value 42 changed to 0:
{ 1 2 3 3 2 1 0 }
